### DISCUSSION TITLE
Closing windows while AltTabbing

### DISCUSSION THREAD

**Discussion by DiagnosiZucchero** [linuxmint/discussions#246] [Ideas]
Something that I do is to open many windows at the same time, to clean the mess afterwards I cycle through them to close duplicates and unused ones, this requires me to press alt+tab, release and press alt+F4 (which I remapped to alt+q for convinience), which if there are many windows is a tedious process which could be streamlined.
Since most windows can be closed without the need to check them fully open, if it was possible to only release tab and keep alt pressed and on top of it press a key like Q or DELETE to close the window that is selected in the alttabbing menu, this would save quite a bit of time. After closing it the alttabbing menu would move to the next window and you could continue cycling by pressing tab as usual and repeat until completion.

In addition, to facilitate this process, it should be possible to resize and make the previews bigger, in order to allow for a faster understanding of the content of each window, and a faster cycling through them, again without releasing alt.

Thanks for the consideration!

**Comment by rcalixte** [comment]
I actually have a similar workflow and what I do instead is trigger the Expo feature (I have a Hot Corner and keybinding configured as triggers). From there, you can 'X' out windows or middle-click them to close them. The Expo view includes minimized windows and also resizes dynamically based on the amount of windows.

**Reply by DiagnosiZucchero** [reply]
It's a nice suggestion and I will use it, but I don't think it works enough for someone that is more keyboard focused.

**Reply by rcalixte** [reply]
You can use the arrow keys to select windows, Ctrl+W to close windows, and Tab to cycle monitors with windows. Enter activates and focuses the window. You can also navigate workspaces using your existing keybindings too.

**Reply by DiagnosiZucchero** [reply]
Quite fancy! It should be more integrated with the default alttab in cinnamon, it has much better functionalities, thanks! (even though it's not as fast as an hypotetical alt+tab/q, just sayin, let me dream)

**Reply by rcalixte** [reply]
Haha, technically, you can change the Keyboard Shortcuts in your settings but that's your call! 😄
